libraries/ms-helper/inc/thermistor.h

```diff
@@ -27,9 +27,8 @@ typedef struct ThermistorStorage {
 StatusCode thermistor_init(ThermistorStorage *storage, GPIOAddress thermistor_gpio,
                            ThermistorPosition position);
 
-// Fetch the temperature reading in milliCelsius from the MCU's ADC
-StatusCode thermistor_get_temp(ThermistorStorage *storage, uint16_t *temperature_millicelcius);
+// Fetch the temperature reading in decicelsius from the MCU's ADC
```

Comment:
Capitalize the C and you should probably add (0.1C) for people who've forgotten their SI units.

Reply:
which C are ypu talking about

Reply:
Celsius

There's only one place it would make any sense to capitalize :P

Reply:
you mean for the variable? Or the comment? 
I thought we used snakecase for variable names?

Reply:
Comment. We do use snake case for variable names.